lib/license_scout/overrides.rb

```diff
@@ -293,10 +293,12 @@ def default_overrides
         ["chef", "Apache-2.0", ["LICENSE"]],
         ["cheffish", "Apache-2.0", ["LICENSE"]],
         ["coderay", nil, ["README_INDEX.rdoc"]],
+        ["coolline", "Zlib", ["LICENSE"]],
```

Comment:
Looking at https://github.com/Mon-Ouie/coolline/blob/master/LICENSE, would this be nil instead of Zlib?